plugin-maven/README.md

```diff
@@ -759,6 +760,42 @@ for details.
 
 <a name="applying-prettier-to-javascript--flow--typescript--css--scss--less--jsx--graphql--yaml--etc"></a>
 
+
+## YAML
+
+- `com.diffplug.spotless.maven.FormatterFactory.addStepFactory(FormatterStepFactory)` [code](https://github.com/diffplug/spotless/blob/main/plugin-maven/src/main/java/com/diffplug/spotless/maven/yaml/Yaml.java)
+
+```xml
+<configuration>
+  <yaml>
+    <includes>   <!-- These are the default includes -->
+      <include>**/*.yaml</include>
+      <include>**/*.yml</include>
```

Comment:
probably a mistake (see [here](https://github.com/diffplug/spotless/pull/1446#issuecomment-1376774346))

Reply:
Yep. I had a doubt with json, and sticked with gradle behavior. I took the initiative for yaml, and asked the change for json. Anyway I'm very comfortable with the strategy you described.